spec/kitchen/driver/base_spec.rb

```diff
@@ -88,6 +88,18 @@ class Slow < Base
     end
   end
 
+  describe ".pre_create_command" do
+    it "run command" do
+      config[:pre_create_command] = "echo works 2&>1 > /dev/null"
```

Comment:
This will fail on Windows. You can use `driver.expects(:run_command).with("echo works 2&>1 > /dev/null")` to avoid actually running anything. Similarly with the fail test, that one is more technically okay but just for unit test speed it would be nice to use Mocha to stub it raising the exception rather than actually running a failed command :)